modular_skyrat/modules/medical/code/smartdarts.dm

```diff
@@ -83,6 +83,11 @@
 		/datum/reagent/medicine,
 		/datum/reagent/vaccine
 	)
+        ///Blacklist that contains medicines that SmartDarts are unable to inject.
+	var/list/disallowed_medicine = list(
+		/datum/reagent/inverse/healing/tirimol,
```

Comment:
Does this even load in? It's already not part of reagent/medicine or reagent/vaccine

Reply:
it does